go/vt/discovery/tablet_stats_cache.go

```diff
@@ -65,8 +68,8 @@ type tabletStatsCacheEntry struct {
 // Note we do the registration in this code to guarantee we call
 // SetListener with sendDownEvents=true, as we need these events
 // to maintain the integrity of our cache.
-func NewTabletStatsCache(hc HealthCheck, cell string) *TabletStatsCache {
-	return newTabletStatsCache(hc, cell, true /* setListener */)
+func NewTabletStatsCache(hc HealthCheck, cell string, ts *topo.Server) *TabletStatsCache {
```

Comment:
One minor comment here: the parameter order should probably be hc, ts, cell (as topo.Server is more general, and cell a specialization). Would match the order of the other method below too.

Reply:
👍  updating PR